packages/flutter/test/widgets/text_golden_test.dart

```diff
@@ -139,4 +140,29 @@ void main() {
       skip: true, // !Platform.isLinux,
     );
   });
+
+  testWidgets('Text Fade', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new RepaintBoundary(
+        child: new MaterialApp(
+          home: new Scaffold(
+            body: new Container(
+              width: 100.0,
+              child: const Text(
+                'Test Test Test Test Test Test Test Test Test Test Test',
+                maxLines: 3,
+                overflow: TextOverflow.fade,
+              ),
+            ),
+          ),
+        )
+      )
+    );
```

Comment:
this should have something in the background so that it's absolutely clear that the fade is being applied only to the text, not to anything else.

Also, maybe use "p" in the text string so that we can see if the fade is being applied only to the text (as it should) or to the whole area where the text might render (including the background, which would be bad).

Reply:
@Hixie : how about the new golden test?

Reply:
You don't need the repaint boundary at the root if you just want to screenshot the whole 800x600 thing. Generally we put the RepaintBoundary on the inside of something that will make the total image smaller (e.g. an Align widget).

Other than that, seems reasonable. Maybe use blue instead of red so that people coming from a reference test background don't think that the test is failing when they see red. :-)

Reply:
Done :smiley: